Starter updates and alphabetical instructs

PeterPeet opened the pull request:
In this pull I have the following:
- Update of the starters
- px change to highlight scrolling in starter window
- Integration of Corpo sub-section in starter
- Added Qwen instruct as separate entry
- Alphabetical sorting of Instruct templates
Made this Pull request mergeable with your main branch.

LostRuins commented:
Sorry, but I'm not really a fan of all the changes in this PR as a whole, to be honest.
- The alphabetical sorting of the instruct formats I do think make sense, we can add those. I can edit them directly or you can PR the changes if you prefer, let me know.
- The modifications to existing scenarios, I don't really like. I think the existing scenarios should remain as they are, except for any potential spelling or grammar fixes if they exist. However, you are welcome to contribute a new scenario if you like, for example the **Summary Assistant**. Also the scenarios are currently somewhat ordered in terms of priority, for example all the "New Document" scenarios and external resources are at the top. 
- I'm not quite sure what opmode=5 is for. Why exactly is it needed? Note that Corpo is a **UI**, not a mode of operation - corpo ui can be used in story and chat mode as well!

PeterPeet commented:
I added some changes that Henky wanted, please let us discuss in https://discord.com/channels/849937185893384223/849938567349796874

PeterPeet commented:
- The alphabetical sorting of the instruct formats I do think make sense, we can add those. I can edit them directly or you can PR the changes if you prefer, let me know.
  - it's not only alphabetical, I also changed two and added one so that **Qwen** is easy to select for someone who doesn't know the correct instruct otherwise (yes I know that is doubled to ChatML, but It's worth it in my opinion to offer **Qwen and Phi-4** as option now that they are new)


- The modifications to existing scenarios, I don't really like. I think the existing scenarios should remain as they are, except for any potential spelling or grammar fixes if they exist.
  - I made only changes where it was necessary. I added the descriptions in the chat names on the buttons, so that it is clear in one view what the genre is.
  - the only scenario I changed is Milton, because there was no real scenario in the character so I added: "Milton is a boy genius and chess prodigy, who can be quite obnoxious. **Can you be his friend?**" so that the players would have a hint what they can do with the character.

- However, you are welcome to contribute a new scenario if you like, for example the Summary Assistant.
  - Yeah, I will add further scenarios, I also improved on the Summary Assistant.
 
- Also the scenarios are currently somewhat ordered in terms of priority, for example all the "New Document" scenarios and external resources are at the top.
  - It is still in order by priority, the only thing thats different is the grouping in the ALL-Section. I have discussed that with Henry and reorganised them, take a look again now I think it makes sense like it is, the most important elements on top.

- I'm not quite sure what opmode=5 is for. Why exactly is it needed? Note that Corpo is a UI, not a mode of operation - corpo ui can be used in story and chat mode as well!
  - Corpo is a drop down that Henky and I wanted to include to create scenarios especially for professional use in the main idea of Corpo-mode as professional UI mode for company use (I named it the same as the UI but a name like "Professional" would have matched)
  - I removed this part and added the elements back into Instruct in the dropdown
